### PR TITLE
Add project description to bootstrap prescription database

### PR DESCRIPTION
**Pull request by fridex**

### CODE REVIEW

File: docs/intern-projects/prescriptions-bootstrap.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+Set of YAML files committed to
+[thoth-station/prescriptions](https://github.com/thoth-station/prescriptions)
+repository that help the recommender system to advise better software stacks to
+users.
+
+## Prerequisites for Team Members
+
+Please check you have all the following:
+
+* Access to Red Hat network
```
Comment (goern) [inline docs/intern-projects/prescriptions-bootstrap.md:22]: is this required due to access to stage?

Comment (fridex) [reply docs/intern-projects/prescriptions-bootstrap.md:22]: It might be not required, will remove it. Thanks. 👍🏻

**Pull request merged**